packages/expo-notifications/plugin/src/withNotificationsAndroid.ts

```diff
@@ -154,9 +180,49 @@ async function removeNotificationIconImageFilesAsync(projectRoot: string) {
   );
 }
 
-export const withNotificationsAndroid: ConfigPlugin = config => {
-  config = withNotificationIconColor(config);
-  config = withNotificationIcons(config);
-  config = withNotificationManifest(config);
+/**
+ * Save sound files to <project-root>/android/app/src/main/res/raw
+ */
+export async function setNotificationSoundsAsync(sounds: string[], projectRoot: string) {
```

Comment:
```suggestion
export async function setNotificationSoundsAsync(projectRoot: string, sounds: string[]) {
```
projectRoot first (matches most of expo tooling), also it seems `{ sounds }` would match other methods here.